services/scan/src/server.ts

```diff
@@ -114,6 +114,9 @@ export async function start({
         importer: resolvedImporter,
         workspace: resolvedWorkspace,
       }));
+
+    // cleanup incomplete batches from before
+    resolvedWorkspace.store.cleanupIncompleteBatches();
```

Comment:
Moving this to the central scanner only because otherwise it would delete the ongoing back with the polls session on VxScan.